src/modules/Accordion/Accordion.js

```diff
@@ -101,24 +101,28 @@ export default class Accordion extends Component {
   renderChildren = () => {
     const { children } = this.props
     const { activeIndex } = this.state
+    let titleIndex = 0
+    let contentIndex = 0
 
-    return Children.map(children, (child, i) => {
+    return Children.map(children, (child) => {
       const isTitle = child.type === AccordionTitle
       const isContent = child.type === AccordionContent
 
       if (isTitle) {
-        const isActive = _.has(child, 'props.active') ? child.props.active : activeIndex === i
+        const currentIndex = titleIndex
```

Comment:
I did that at first

```
if (isTitle) {
  const isActive = _.has(child, 'props.active') ? child.props.active : activeIndex === titleIndex
  const onClick = (e) => {
    this.handleTitleClick(e, titleIndex)
    if (child.props.onClick) child.props.onClick(e, titleIndex)
  }
  titleIndex++
  return cloneElement(child, { ...child.props, active: isActive, onClick })
}

if (isContent) {
  const isActive = _.has(child, 'props.active') ? child.props.active : activeIndex === contentIndex
  contentIndex++
  return cloneElement(child, { ...child.props, active: isActive })
}
```

But if you do this, than all titles will have index 3 in the callback. Like the lambda gets not only lazy evaluation, but lazy creation.